NorthstarDedicatedTest/bansystem.cpp

```diff
@@ -40,6 +59,14 @@ void ServerBanSystem::ClearBanlist()
 
 void ServerBanSystem::BanUID(uint64_t uid)
 {
+	// checking if last char is \n to make sure uids arent getting fucked
+	std::ifstream fsBanlist(GetNorthstarPrefix() + "/banlist.txt");
+	std::string content((std::istreambuf_iterator<char>(fsBanlist)), (std::istreambuf_iterator<char>()));
+	fsBanlist.close();
+
+	if (content.back() != '\n')
+		m_sBanlistStream << std::endl;
+
```

Comment:
Soooo that didn't work in testing :/

Reply:
Had like an commented out UID in file with no newline

```
# AAAAAA
```

and on ban it looked like
```
# AAAAAABBBBBB
```

Reply:
Nevermind, issue was on my end. Banlist only writes `\n` but my text editor expected `\r\n`, resulting in it not showing line breaks. We can talk about switching to `\r\n` but that should be a separate PR considering that current pre-PR implementation also only just wrote `\n`.

Reply:
silly little windows 8 user

Reply:
:trollface: